### DISCUSSION TITLE
Python jupyter notebook

### DISCUSSION THREAD

**Discussion by vnom1** [coder/code-server#3556] [Q&A] [answered]
Hi,
I deployed code-server on Railway

 
Code-server is amazing,but have some problems in extension, like

+ jupyter notebook 
+ bracket pair colorize 
+ Sometimes ms python

Thx in advance

**Comment by vital987** [comment, marked as answer]
Use [Bracket Pair Colorizer 2](https://marketplace.visualstudio.com/items?itemName=CoenraadS.bracket-pair-colorizer-2), It worked for me, also check out my [project](https://github.com/vital987/code-server) in free time.

**Reply by vnom1** [reply]
Yay, I saw ur repo 
It's gud

**Reply by vital987** [reply]
What problems are you facing with Jupyter and Python extensions?
